indico/util/spreadsheets.py

```diff
@@ -86,17 +88,19 @@ def generate_csv(headers, rows):
     return buf
 
 
-def _prepare_excel_data(data):
+def _prepare_excel_data(data, tz=None):
     if isinstance(data, (list, tuple)):
         data = ', '.join(data)
     elif isinstance(data, set):
         data = ', '.join(sorted(data, key=unicode.lower))
     elif is_lazy_string(data) or isinstance(data, Markup):
         data = unicode(data)
+    elif isinstance(data, datetime.datetime) and tz:
+        data = format_datetime(data, timezone=tz)
```

Comment:
`.decode('utf-8')` since these functions return UTF-8 but everything else here is unicode.